packages/legend-studio/src/stores/editor-state/element-editor-state/RuntimeEditorState.ts

```diff
@@ -275,20 +297,23 @@ export const getRuntimeExplorerTreeData = (
               ) {
                 return connectionValue.class.value;
               }
-              throw new UnsupportedOperationError();
+              return undefined;
             })
             .concat(allSourceClassesFromMappings),
         ); // make sure we add classes (from mappings) that we expect to have connections for
         // classes (2nd level) - only for `ModelStore`
-        classes.forEach((_class) => {
-          const classNode = {
-            data: _class,
-            id: _class.path,
-            label: _class.name,
-          };
-          nodes.set(classNode.id, classNode);
-          addUniqueEntry(childrenIds, classNode.id);
-        });
+        classes
+          .filter((_class) => _class !== undefined)
```

Comment:
does `.filter(Boolean)` or `.filter(isNonNullable)` work, so we don't have to do `const sourceClass = guaranteeNonNullable(_class);`?